### PR TITLE
Made the changes according to the guidelines to the bottom sheet example app

### PR DESCRIPTION
**Pull request by Rishika-7**
Made the changes according to the guidelines, the package name and the branch name to the bottom sheet example app

### CODE REVIEW

File: bottom_sheet/android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.project"
```
Comment (nisrulz) [inline bottom_sheet/android/app/build.gradle:41]: The package name is not fixed still 🙄

File: bottom_sheet/android/app/src/main/kotlin/com/example/project/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.example.project
```
Comment (nisrulz) [inline bottom_sheet/android/app/src/main/kotlin/com/example/project/MainActivity.kt:1]: Same here.

**Comment by Rishika-7** [comment]
Corrected the package name in all the files

File: bottom_sheet/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# project
+
+A new Flutter project.
```
Comment (nisrulz) [inline bottom_sheet/README.md:3]: Provide description and demo gif of the example app. Check other example apps for reference.

File: bottom_sheet/test/widget_test.dart [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    expect(find.text('1'), findsNothing);
+
+    // Tap the '+' icon and trigger a frame.
+    await tester.tap(find.byIcon(Icons.add));
+    await tester.pump();
+
+    // Verify that our counter has incremented.
+    expect(find.text('0'), findsNothing);
+    expect(find.text('1'), findsOneWidget);
+  });
```
Comment (nisrulz) [inline bottom_sheet/test/widget_test.dart:29]: Write proper tests.

File: bottom_sheet/lib/main.dart [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+void _bottomSheetMore(context) {
+  showModalBottomSheet(
+    context: context,
+    builder: (builder) {
+      return new Container(
+        padding: EdgeInsets.only(
+          left: 5.0,
+          right: 5.0,
+          top: 5.0,
+          bottom: 5.0,
```
Comment (nisrulz) [inline bottom_sheet/lib/main.dart:65]: For common values, use a variable.

File: bottom_sheet/lib/main.dart [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        ),
+        decoration: new BoxDecoration(
+            color: Colors.white,
+            borderRadius: new BorderRadius.only(
+                topLeft: const Radius.circular(10.0),
+                topRight: const Radius.circular(10.0))),
```
Comment (nisrulz) [inline bottom_sheet/lib/main.dart:71]: For common values, use a variable.

File: bottom_sheet/lib/main.dart [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+            new ListTile(
+              title: const Text(
+                'Logout',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+              onTap: () async {
+                // Add Here
```
Comment (nisrulz) [inline bottom_sheet/lib/main.dart:146]: Noting happens when you click on the items in Bottom Sheet. Ideally it should do something, for the least it should dismiss the Bottom Sheet.

File: bottom_sheet/lib/main.dart [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+                ),
+              ),
+              title: const Text(
+                'Favourites',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ),
```
Comment (nisrulz) [inline bottom_sheet/lib/main.dart:99]: Each `ListTile`'s code is same. Create a function to abstract that and pass in arguments to the function so you can construct a `ListTile`.

```
ListTile getListTile(icon, iconColor, titleText) {
  return new ListTile(
    leading: new Container(
      width: 4.0,
      child: Icon(
        icon,
        color: iconColor,
        size: 24.0,
      ),
    ),
    title: new Text(
      titleText,
      style: TextStyle(
        fontSize: 14.0,
        fontWeight: FontWeight.w700,
      ),
    ),
  );
}
```

and then use as:

```
 children: <Widget>[
            getListTile(Icons.favorite, Colors.pink, "More"),
            getListTile(Icons.favorite, Colors.pink, "Favourites")
            ]
```

File: bottom_sheet/lib/main.dart [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+      theme: ThemeData(
+        primarySwatch: Colors.red,
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+      ),
+      home: MyHomePage(title: 'Bottom Sheet'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
```
Comment (nisrulz) [inline bottom_sheet/lib/main.dart:22]: Split your code into files.

**Review by nisrulz — CHANGES_REQUESTED** [review]
Please add a link of example app in the repo readme file's list.

Also address the comments in the PR.

**Comment by nisrulz** [comment]
Also add a demo gif in the example directory and reference in readme. Check other example apps for reference.

**Comment by Rishika-7** [comment]
I have made the changes according to the comments.

**Comment by nisrulz** [comment]
Looking better than last time. Good work @Rishika-7 ! 

I will give it a run later today.

Thank you

**Comment by Rishika-7** [comment]
Thank you I have learnt a lot! Hoping it's upto the mark :)

**Comment by nisrulz** [comment]
@Rishika-7 everything looks good to me. I noticed 2 issues:

1. Inside `home.dart` file:
Screenshot 2020-10-08 at 02 50 26

Solution will be to change the icon to `Icons.exit_to_app`.

2. The screenshot for demo is placed under `assets/`, which is wrong as it not a asset for the app but rather a demo gif file.
Screenshot 2020-10-08 at 02 53 26

If you look at other examples in the repo, you will see the demo gif just sits in the root directory of the example app itself and is named `demo.gif`.  I would suggest to follow the same and also update the link in all readme, where you reference this gif.

Rest everything works!

**Comment by Rishika-7** [comment]
I have made the changes!

**Comment by nisrulz** [comment]
Great!

I think this PR is ready to merged.  Thank you for your contribution 🎉

**Review by nisrulz — APPROVED** [review]

**Pull request merged**